spec/models/user_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe User do
+    before(:each) { @user = User.new(email: 'user@example.com') }
```

Comment:
let(:user) { User.new(email: 'user@example.com')
And then call user instead of @user